### PR TITLE
few fixes

### LINKED ISSUE #29: new app not updated in app list.
list doesn't update when i install a new app

### PR DESCRIPTION
**Pull request by sherlockbeard**
#29 #28 #18 are resolved in this request

### CODE REVIEW

**Comment by F4uzan** [comment]
Just as a future reference: I suggest splitting commits if the changes are irrelevant to one another. It's alright to lump them, but it'll make reverting a tad hard sometimes.

That aside, I'll be testing this out soon. Thanks!

**Comment by sherlockbeard** [comment]
@F4uzan sorry it's my first PR.

**Comment by F4uzan** [comment]
It's alright! It's not a big PR, so we can manage with them in one commit for now.

File: app/src/main/java/mono/hg/LauncherActivity.java [hunk 1]
```diff
@@ -938,7 +938,8 @@ private void addPanelListener() {
 
             @Override public void onPanelStateChanged(View panel, int previousState, int newState) {
                 if (newState == SlidingUpPanelLayout.PanelState.COLLAPSED
-                        || newState == SlidingUpPanelLayout.PanelState.DRAGGING) {
+                        /*|| newState == SlidingUpPanelLayout.PanelState.DRAGGING*/) {
```
Comment (F4uzan) [inline app/src/main/java/mono/hg/LauncherActivity.java]: This does work, but causes the search bar to appear late. We can probably add a boolean somewhere to better its use.

File: app/src/main/java/mono/hg/fragments/HiddenAppsFragment.java [hunk 2]
```diff
@@ -106,7 +106,8 @@ private void loadApps() {
         Collections.sort(availableActivities, new ResolveInfo.DisplayNameComparator(manager));
 
         // Clear the list to make sure that we aren't just adding over an existing list.
-        appList.clear();
+        // hit and try solution for new apps not adding in the list. didn't execpted how but it worked can try.
```
Comment (F4uzan) [inline app/src/main/java/mono/hg/fragments/HiddenAppsFragment.java]: This is unrelated to the app list. HiddenAppsFragment _always_ fetch the latest list of apps (thereby not requiring a receiver to listen for any new apps). Removing this line serves very little use.

File: app/src/main/res/values/strings_settings.xml [hunk 3]
```diff
@@ -66,8 +66,8 @@
     <string name="windowbar_mode_nav">Hide navigation bar</string>
     <string name="windowbar_mode_both">Hide both</string>
 
-    <string name="gestures_left">Swipe left</string>
-    <string name="gestures_right">Swipe right</string>
+    <string name="gestures_left">Swipe &lt;---</string>
+    <string name="gestures_right">Swipe ---&gt;</string>
```
Comment (F4uzan) [inline app/src/main/res/values/strings_settings.xml]: I'm still troubled by the string here. Would this make it more understandable? We may be using a better indication here, however — and that may be enough.

File: app/src/main/java/mono/hg/LauncherActivity.java (hunk 1 above)
Comment (sherlockbeard) [reply app/src/main/java/mono/hg/LauncherActivity.java]: ok. i will try some other ways today.

File: app/src/main/res/values/strings_settings.xml (hunk 3 above)
Comment (sherlockbeard) [reply app/src/main/res/values/strings_settings.xml]: ok i am not sure how to talk back commit or unchanged can you discard this file .

File: app/src/main/res/values/strings_settings.xml [hunk 4]
```diff
@@ -66,8 +66,8 @@
     <string name="windowbar_mode_nav">Hide navigation bar</string>
     <string name="windowbar_mode_both">Hide both</string>
 
-    <string name="gestures_left">Swipe left</string>
-    <string name="gestures_right">Swipe right</string>
+    <string name="gestures_left">Swipe Left</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings_settings.xml]: I much prefer ← and →, and "up" is not consistent if this is "Left". I don't think it qualifies as a headline.

**Comment by F4uzan** [comment]
Due to general inactivity, I'll be closing this PR. Feel free to reopen it (or do another PR) if you think the changes here are necessary.
